src/main/scala/com/socrata/geospace/lib/regioncache/HashMapRegionCache.scala

```diff
@@ -54,4 +55,13 @@ class HashMapRegionCache(config: Config) extends MemoryManagingRegionCache[Map[S
       .collect { case (key: RegionCacheKey, Some(Success(index))) => (key, index.size) }
       .sortBy(_._2)
       .reverse
+
+  /**
+   * Returns indices in descending order of Age, not in constant time
+   * @return Indices in descending order of Age
+   */
+  override def keysByLeastRecentlyUsed(): Seq[RegionCacheKey] = {
+    cache.ascendingKeys().toSeq.asInstanceOf[Seq[RegionCacheKey]]
```

Comment:
Do cache entries have a public property like `lastAccessed`? If this is how we're going to depressurize the cache going forward, it could be helpful to return this information and then update the region-coder debug endpoint to use this method instead of `indicesBySizeDesc`.